test/e2e/setup.go

```diff
@@ -73,6 +83,156 @@ func skipIfNotInDevelopmentEnv() {
 	}
 }
 
+func TakeScreenshot(wd WebDriver, e error) {
+	log.Info("Taking Screenshot and saving page source")
```

Comment:
this is cool, do you have any example runs with the screenshots still lingering around to look at?

Reply:
The builds have expired unfortunately... But in the APAC Demos I did show off what it looks like in ADO if you wanted to see that https://drive.google.com/file/d/1wvdk6CfWxY7K_9rNvhH90lyNg1dURH9-/view?usp=sharing 

It starts at 25:10

Reply:
awesome, thank you!